### PR TITLE
add new developer portfolio

### PR DESCRIPTION
**Pull request by abdellatif-laghjaj**
In this change, I add my own developer portfolio to the existing list.

### CODE REVIEW

**Comment by eclectic-coding** [comment]
Thank you for your interest. Can you edit your listing to be in alphabetical order based on your first name? Then I will merge.

**Comment by abdellatif-laghjaj** [comment]
Yes i already add it in the right order

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 - [Aashutosh Rathi](https://aashutosh.dev)
 - [Aayush Kurup](https://aayushkurup.dev)
 - [AbdeNassar Amimi](https://abdenassar-portfolio-4smfcqph6-abdenassaramimi99-gmailcom.vercel.app/)
+- [Abdellatif Laghjaj](http://abdellatif-laghjaj.ml/)
 - [Abdelaziz El Arassi](http://aelarassi.com/)
```
Comment (eclectic-coding) [inline README.md:27]: You need to add your listing below `Abdelaziz El Arassi`. Thanks.

Comment (abdellatif-laghjaj) [reply README.md]: Done

**Pull request merged**